<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white; /* color de fondo */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Vista/grande.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnComprobante">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Comprobante Digital</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVerCliente">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Ver Como Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReporte">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Reporte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInventario">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistroProducto">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Registro de Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultaProveedores">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Consulta de Proveedores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultaCliente">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Consulta de Clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegresarMenu">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #8000FF; /* morado */
    color: white;              /* texto blanco */
    font-weight: bold;
    border-radius: 5px;        /* esquinas redondeadas */
    height: 40px;
}
QPushButton:hover {
    background-color: #a64dff; /* morado m√°s claro al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;       /* negro */
font-weight: bold;     /* negrita */
font-family: Arial;    /* opcional, tipo de letra */
font-size: 9pt;       /* opcional, tama√±o */</string>
    </property>
    <property name="text">
     <string>Version 2025.1.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#ffffff;&quot;&gt;Admin.Jim-Zu√±i-01&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>189</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: black;            /* negro */
    font-weight: bold;        /* negrita */
    font-size: 16pt;          /* tama√±o uniforme */
    qproperty-alignment: AlignCenter; /* centrado */
}
</string>
    </property>
    <property name="text">
     <string>BIENVENIDO AL SISTEMA BODEGA BYTE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>168</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Administrador üõ°Ô∏è 09/11/2025&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>btnComprobante</zorder>
   <zorder>label</zorder>
   <zorder>btnVerCliente</zorder>
   <zorder>btnReporte</zorder>
   <zorder>btnInventario</zorder>
   <zorder>btnRegistroProducto</zorder>
   <zorder>btnConsultaProveedores</zorder>
   <zorder>btnConsultaCliente</zorder>
   <zorder>btnRegresarMenu</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../lu.qrc"/>
 </resources>
 <connections/>
</ui>
